### PR TITLE
Users/shmittal/just check

### PR DESCRIPTION
**Pull request by 20shivangi**

### CODE REVIEW

File: images/linux/Ubuntu1604-README.md [hunk 1]
```diff
@@ -200,6 +200,8 @@ the
 - Android ConstraintLayour 1.0.1
 - Az Module (1.0.0)
 - Az Module (1.6.0)
+- Az Module (2.3.2)
```
Comment (niadak) [inline images/linux/Ubuntu1604-README.md]: Should we call out that this modules are not installed and is saved in the specific path. User needs to update the PSModulePath to use them?

Comment (niadak) [reply images/linux/Ubuntu1604-README.md]: There is a open issue for the same https://github.com/microsoft/azure-pipelines-image-generation/issues/1208

Comment (20shivangi) [reply images/linux/Ubuntu1604-README.md]: Done

**Comment by 20shivangi** [comment]
@kaylangan Can you tell me what I should do to update the README file ?

**Pull request closed without merging**